tests/e2e/basic_test.go

```diff
@@ -193,8 +194,13 @@ func (s *BasicTestSuite) TestVmGroupVolumeIsolation(c *C) {
 	out, err = dockercli.DeleteVolume(s.vm1, s.volName2)
 	c.Assert(err, IsNil, Commentf(out))
 
+	// Clean up the vm group
+	out, err = admincli.DeleteVMgroup(s.esx, vmgroup)
+	c.Assert(err, IsNil, Commentf(out))
+
```

Comment:
Let's not mix different issues together. We already have an issue for that: https://github.com/vmware/docker-volume-vsphere/issues/1411